solutionbox/structured_data/mltoolbox/_structured_data/_package.py

```diff
@@ -59,15 +59,7 @@ class FileNotFoundError(IOError):
 
 def _default_project():
   from google.datalab import Context
-  return Context.default().project_id
-
-
-def _is_in_IPython():
-  try:
-    import IPython # noqa
-    return True
-  except ImportError:
-    return False
+  return Context.default().project_id.decode()
```

Comment:
If the `.decode()` is for python compatibility we could build a helper function that will automatically apply the `.decode()` if python 3.